zhaquirks/xiaomi/custom/z03mmc.py

```diff
@@ -0,0 +1,153 @@
+            type=int16s,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+
+
+class RelativeHumidityCustom(CustomCluster, RelativeHumidity):
+    """Relative Humidity Cluster with calibration attribute."""
+
+    class AttributeDefs(RelativeHumidity.AttributeDefs):
+        """Attribute Definitions."""
+
+        # A value in 0.01%RH offset to fix up incorrect values from sensor
+        humidity_calibration = ZCLAttributeDef(
+            id=0x0010,
+            type=int16s,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+
+
+class UserInterfaceCustom(CustomCluster, UserInterface):
+    """Custom User Interface Cluster with smiley control."""
+
+    class AttributeDefs(UserInterface.AttributeDefs):
+        """Attribute Definitions."""
+
+        # of the 3 ZCL Thermostat User Interface spec attributes,
+        # only the first one (TemperatureDisplayMode) is implemented fully.
+        # KeypadLockout is implemented but completely unused in the device firmware
+        # and ScheduleProgrammingVisibility is not implemented at all
+        # https://github.com/devbis/z03mmc/blob/1.1.0/src/sensorEpCfg.c#L256
+
+        # 0 - smiley is off, 1 - smiley is on (according to comfort values)
+        smiley = ZCLAttributeDef(
+            id=0x0010,
+            type=Bool,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+
+        # display. 0 - display is off, 1 - display is on
+        display = ZCLAttributeDef(
+            id=0x0011,
+            type=Bool,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+
+        # comfort temperature min: A value in 0.01ºC to set minimum comfort temperature for happy face
+        comfort_temperature_min = ZCLAttributeDef(
+            id=0x0102,
+            type=int16s,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+
+        # comfort temperature max: A value in 0.01ºC to set maximum comfort temperature for happy face
+        comfort_temperature_max = ZCLAttributeDef(
+            id=0x0103,
+            type=int16s,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+
+        # comfort humidity min: A value in 0.01%RH to set minimum comfort humidity for happy face
+        comfort_humidity_min = ZCLAttributeDef(
+            id=0x0104,
+            type=uint16_t,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+
+        # comfort humidity max: A value in 0.01%RH to set maximum comfort humidity for happy face
+        comfort_humidity_max = ZCLAttributeDef(
+            id=0x0105,
+            type=uint16_t,
+            access="rw",
+            is_manufacturer_specific=True,
+        )
+
+
+# https://github.com/devbis/z03mmc
+# defined by 1.1.0 firmware (0x11003001)
+# see README.md in the repo for more info
+class LYWSD03MMC_devbis(XiaomiCustomDevice):
```

Comment:
Also, we generally don't want to use underscores in class names. I see why you did it here, but maybe there could be a better name.